### ISSUE TITLE
Request: Make "Use Head Direction for movement" default to enabled

### ISSUE THREAD

**Issue by StarfishHidari** [Frooxius/NeosPublic#1237]
A large majority of players new to Neos or vr in general do not use hand direction for movement and frequently find it more difficult to figure out locomotion in comparison to locomotion using head direction for movement. When I'm introducing new players to Neos I always have to explain to them how to turn on head direction for movement so that they have an easier time with locomotion.

Making "Use Head Direction for movement" default to enabled would be a default that is much more accommodating to new users.

**Comment by StarfishHidari** [comment]
(i realized this is already default)

**Comment by StarfishHidari** [comment]
I've been bumping into a lot of new users who are still very confused by hand orientation being default, I had closed this originally thinking that it was default based on what a few players in game had told me at the time, but I think they had already set that themselves and just forgot.

**Comment by H3BO3** [comment]
https://discord.com/channels/402159838827905024/469131434628612116/728953744448028793 This was set to default back in August. Do you think it's a regression? Bear in mind it will only affect brand new accounts.

**Comment by shiftyscales** [comment]
Are you certain these are new accounts, @StarfishHidari? Could they be users revisiting the game/with existing accounts created from before? Do you have any example usernames of users that have had this issue?

**Comment by H3BO3** [comment]
I did a check on an account I created a month or so ago, Use Head Direction for movement is NOT set as default. Not sure what happened between July and now, but it's not set as such.

**Comment by shiftyscales** [comment]
@H3BO3 Frooxius noted to me that if you used the same local database, the value likely carried through.

@StarfishHidari I just checked on a fresh local database and the head direction is the default. I did however discover that the left controller is set to the primary controller by default which would cause the sticks to be swapped so the right controller strafes- which could be another potential source of confusion among users. That can be tweaked though.
